src/class/cdc/cdc_device.c

```diff
@@ -128,6 +128,29 @@ void tud_cdc_n_set_wanted_char (uint8_t itf, char wanted)
   _cdcd_itf[itf].wanted_char = wanted;
 }
 
+uint32_t tud_cdc_n_abort_transfer (uint8_t itf)
+{
+  cdcd_interface_t* p_cdc = &_cdcd_itf[itf];
+
+  // Skip if usb is not ready yet
+  TU_VERIFY( tud_ready(), 0 );
+
+  uint8_t const rhport = TUD_OPT_RHPORT;
+
+  // Claim the endpoint, if not marked busy
+  usbd_edpt_claim(rhport, p_cdc->ep_in);
+
+  // Stall the endpoint
+  usbd_edpt_stall(rhport, p_cdc->ep_in);
+
+  // Clear the stalled endpoint
+  usbd_edpt_clear_stall(rhport, p_cdc->ep_in);
+
+  // Release the endpoint
```

Comment:
The function name clearly describes what is the purpose so need for duplication by comments. 
```suggestion
```